frontend/model/contracts/chatroom.js

```diff
@@ -66,7 +66,7 @@ function addMention ({ contractID, messageId, datetime, text, username, chatRoom
   makeNotification({
     title: `# ${chatRoomName}`,
     body: text,
-    icon: rootGetters.globalProfile2(groupID, username).picture,
+    icon: rootGetters.globalProfile2(groupID, username)?.picture,
```

Comment:
@Silver-IT I forgot to add - we need a comment above these lines explaining why this `?.` is here, to prevent someone from removing it in the future.

Something like:

```js
// this ?. is important - as in some scenarios the profile could be undefined here, see issue #1420
```

Maybe this could be added in a future PR you send?

Reply:
Yeah. I will add this comment in the next PR...